libs/api/data-transfer/src/lib/models/notification.model.ts

```diff
@@ -0,0 +1,16 @@
+import { UserModel } from './user.model';
+
+export enum NotificationType {
+  STATUS_CHANGED = 'ORDER_STATUS_CHANGED',
```

Comment:
We're using PascalCase in enums. Keep in mind value of this enum will be stored in database

```
OrderStatusChanged = 'ORDER_STATUS_CHANGED'
```